src/psyclone/psyir/transformations/extract_trans.py

```diff
@@ -57,13 +57,22 @@ class ExtractTrans(RegionTrans):
     Nodes to extract can be individual constructs within an Invoke (e.g. \
     Loops containing a Kernel or BuiltIn call) or entire Invokes. This \
     functionality does not support distributed memory.
+
+    :param node_class: The Node class of which an instance will be inserted \
+        into the tree (defaults to ExtractNode).
+    :type node_class: :py:class:`psyclone.psyir.nodes.ExtractNode`
```

Comment:
Strictly-speaking, I think the type of node_class is `type`. At least, that's what pdb will (probably) tell you. Or, will it always be a sub-class of ExtractNode - I guess since this is ExtractTrans then that will be the case? Please clarify a bit in the docstring.

Reply:
While in theory any class could be specified in the transform, de-facto only ExtractNodes are used (including classes derived from ExtractNode). I've reworded the documentation to make this clearer.

Reply:
Thanks. You're missing a couple of line-continuation slashes.

Reply:
Dito